examples/models/transform_jitter.py

```diff
@@ -1,3 +1,18 @@
+"""
+==========================================
+Jittered Scatter Plot with Bokeh and NumPy
+==========================================
+
+This example demonstrates how to add jitter to a scatter plot using Bokeh and NumPy.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.circle, bokeh.models.sources.ColumnDataSource, bokeh.models.markers.Circle, bokeh.models.Jitter, bokeh.models.LabelSet, bokeh.models.widgets.Button, bokeh.models.callbacks.CustomJS
+    :refs: :ref:`userguide_plotting` > :ref:`userguide_plotting` > :ref:`userguide_widgets`, :ref:`userguide_styling` > :ref:`userguide_annotations`
```

Comment:
These lines are too long (this is also why the CI is failing, https://github.com/bokeh/bokeh/actions/runs/4404874372/jobs/7715229116), please add some line breaks to stay below the max of 165 characters per line!